pymc3/tests/test_distributions_random.py

```diff
@@ -739,6 +744,18 @@ def ref_rand(size, alpha, mu):
     def test_geometric(self):
         pymc3_random_discrete(pm.Geometric, {"p": Unit}, size=500, fails=50, ref_rand=nr.geometric)
 
+    def test_hypergeometric(self):
+        def ref_rand(size, N, k, n):
+            return st.hypergeom.rvs(M=N, n=k, N=n, size=size)
+
+        pymc3_random_discrete(
+            pm.HyperGeometric,
+            {"N": Nat, "k": Nat, "n": Nat},
+            size=500,
+            fails=50,
+            ref_rand=ref_rand,
+        )
```

Comment:
```suggestion
        pymc3_random_discrete(
            pm.HyperGeometric,
            {"N": Domain([10, 11, 12, 13], "int64"),
             "k": Domain([4, 5, 6, 7], "int64"),
             "n": Domain([6, 7, 8, 9], "int64")},
            size=500,
            fails=50,
            ref_rand=ref_rand,
        )
```
You can create your own domain like this.

Reply:
This should avoid the error that you are facing

Reply:
Specifying domain fixed it. Wonderful, thanks!